editor/svg-editor.js

```diff
@@ -2037,20 +2041,6 @@ TODOS
 							// Clicking the icon in flyout should set this set's icon
 							var func = function(event) {
 								var options = opts;
-								//find the currently selected tool if comes from keystroke
-								if (event.type === 'keydown') {
-									var flyoutIsSelected = $(options.parent + '_show').hasClass('tool_button_current');
-									var currentOperation = $(options.parent + '_show').attr('data-curopt');
-									$.each(holders[opts.parent], function(i, tool) {
-										if (tool.sel == currentOperation) {
-											if (!event.shiftKey || !flyoutIsSelected) {
-												options = tool;
-											} else {
-												options = holders[opts.parent][i+1] || holders[opts.parent][0];
-											}
-										}
-									});
-								}
```

Comment:
Ok, I played around with it on the trunk and it seems that pressing L selects the line tool but it doesn't show up in the toolbar as selected, while pressing R selects the rectangle tool and it shows as selected.  I guess you are reasonably sure this doesn't break anything so let's go for it.